go.php

```diff
@@ -4,27 +4,27 @@
 ############## Adjust when editing docs ##################
 
 $mapping = array(
-    'admin-background-jobs'   => '/admin_manual/configuration_server/background_jobs_configuration.html',
+    'admin-background-jobs'   => '/admin_manual/configuration/server/background_jobs_configuration.html',
     'admin-dir_permissions'   => '/admin_manual/installation/installation_wizard.html#setting-strong-directory-permissions',
-    'admin-encryption'        => '/admin_manual/configuration_files/encryption_configuration.html',
-    'admin-external-storage'  => '/admin_manual/configuration_files/external_storage_configuration_gui.html',
+    'admin-encryption'        => '/admin_manual/configuration/files/encryption_configuration.html',
+    'admin-external-storage'  => '/admin_manual/configuration/files/external_storage_configuration_gui.html',
     'admin-install'           => '/admin_manual/installation/index.html',
-    'admin-ldap'              => '/admin_manual/configuration_user/user_auth_ldap.html',
-    'admin-provisioning-api'  => '/admin_manual/configuration_user/user_provisioning_api.html',
-    'admin-sharing'           => '/admin_manual/configuration_files/file_sharing_configuration.html',
-    'admin-sharing-federated' => '/admin_manual/configuration_files/federated_cloud_sharing_configuration.html',
+    'admin-ldap'              => '/admin_manual/configuration/user/user_auth_ldap.html',
+    'admin-provisioning-api'  => '/admin_manual/configuration/user/user_provisioning_api.html',
+    'admin-sharing'           => '/admin_manual/configuration/files/file_sharing_configuration.html',
+    'admin-sharing-federated' => '/admin_manual/configuration/files/federated_cloud_sharing_configuration.html',
     'admin-source_install'    => '/admin_manual/installation/source_installation.html',
     'admin-backup'            => '/admin_manual/maintenance/backup.html',
     'admin-monitoring'        => '/admin_manual/operations/considerations_on_monitoring.html',
```

Comment:
It was also removed from core with in https://github.com/owncloud/core/pull/28416 and https://github.com/owncloud/core/pull/28417 so everything fine.